data/human/weapons.txt

```diff
@@ -1470,8 +1470,8 @@ outfit "Nuclear Missile"
 		"optical tracking" 1
 		"trigger radius" 30
 		"blast radius" 150
-		"shield damage" 9000
-		"hull damage" 7000
+		"shield damage" 4500
+		"hull damage" 3500
```

Comment:
Do we really want to change this? Nobody use them as-is.

Reply:
At least increase the blast radius mb?

Reply:
> At least increase the blast radius mb?

I'd rather avoid mixing this up with different changes like that. This PR is for rescaling the tier scaling of damage, if the nuke needs to be buffed relative to other human weapons that should be a separate PR.

Reply:
I would suggest leaving the nuclear missile stats unchanged. It is supposed to be utterly terrifying with a high chance of one-shotting many human ships.

While I agree this isn't the time or place to make changes to it to make it more effective, I see no problem with merely leaving it untouched. Regarding balancing it versus other human weapons, it really has no real comparable weapon to contrast with, since no other missile is a single shot taking up a gun port. It isn't used much because it's isn't a weapon suitable for fighting with. It is a weapon for intimidation, genocide, and a strategic warfare of the type that happily sacrifices a ship carrying these weapons to take out fleets. While FW, Navy, and Syndicate Admirals all heartily approve of sacrificing a few small spots to take out a fleet, it is of really low utility to the player for obvious reasons.

Tldr. Leave the nuke as is. It is appropriate to leave as is and balancing is not a concern since it isn't really a relevant weapon anyway.

Reply:
Tbf we're trying to slow down combat, and it is the highest dps weapon, and on top of that isnt actually used in battles, besides against the player (it definitely killed me a bunch)

Reply:
I too think the nuke should remain unchanged.
Keeping the nuke's damage as is would also by comparison make the Checkmate branch more rewarding and appealing than it currently is, at least in the sense that it'd be a clearly far better option in the human arsenal.

Reply:
lol. I suppose another way to put it, in a discussion about slowing down battles, is to point out that the nuclear missile is the unique, solitary human weapon that is supposed to generate battles that are super short, terrifying, and devastatingly deadly. (I mean, all weapons want to do that, but this is the one weapon where story usage and meta dictates that it has actually achieved that goal)

It is, according to our lore, the only weapon in human space so powerful that every single ship in existence (including aliens, somehow) has a built-in warning light in the cockpit to warn the Captain when a ship carrying these weapons enters the system. 

How our ship knows, I don't know. Space is notoriously filled with radiation, so my guess is our FTL LIDAR that runs our minipap and targeting system can recognize a nuclear missile protruding from a gunport.